bin/init.js

```diff
@@ -351,20 +573,35 @@ const getRoot = () => {
  */
 const runThemeCleanup = () => {
 	const deleteDirs = [
-		'.git',
 		'.github',
-		'bin',
+		'bin/init.js',
 		'languages',
 	];
 
+	if ( ! isGitInitialized ) {
+		deleteDirs.push( '.git' );
+	}
+
 	deleteDirs.forEach( ( dir ) => {
 		const dirPath = path.resolve( getRoot(), dir );
 		try {
 			if ( fs.existsSync( dirPath ) ) {
-				fs.rmdirSync( dirPath, {
+				let isDirectory = false;
+
+				if ( true === fs.lstatSync( dirPath ).isDirectory() ) {
+					isDirectory = true;
+				}
+
+				// rmSync function introduced in Node v14.14.0. It can delete files and directories recursively.
```

Comment:
do we need this comment?

Reply:
There is a function called `rmdirSync` which gave me a deprecation warning when tried to use it here. This is the alternative for that function so added explanation and use of this function for other developers that this function will give the same results.